apps/ledger-live-desktop/src/renderer/modals/SignMessage/Components/CopyButton.test.tsx

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import { render, screen, fireEvent, waitFor } from "@testing-library/react";
```

Comment:
You can use 
```suggestion
import { render, screen, fireEvent, waitFor } from "tests/testUtils";
```

You will not need to wrap and you can remove 

`import { DefaultTheme, ThemeProvider } from "styled-components";`